docs/docs/alephium/collections.md

```diff
@@ -0,0 +1,595 @@
+
+| Type                   | Key                               | Value                                 |
+| ---------------------- | --------------------------------- | ------------------------------------- |
+| mapping[ByteVec, TickmapBatch] | The pool key of a specified pool and `TickmapBatch` index. | TickmapBatch struct holding the chunks' data. |
+
+### Get Tick's position
+
+```rust
+pub fn tickToPosition(tick: I256, tickSpacing: U256) -> (U256, U256);
+```
+
+Calculates where tick's bit will be located.
+
+#### Input parameters
+
+| Name  | Type    | Description           |
+| ----- | ------- | --------------------- |
+| tick   | I256 | Index of a tick.|
+| tickSpacing | U256     | The spacing between initializable ticks.|
+
+#### Output parameters
+
+| Type | Description                 |
+| ---- | --------------------------- |
+| U256 | Index of the chunk. |
+| U256 | Index of the bit in a chunk. |
+
+
+### Get Chunk
+
+```rust
+fn getChunk(chunk: U256, poolKey: PoolKey) -> U256;
+```
+
+Retrieve chunk's data.
+
+| Name  | Type    | Description           |
+| ----- | ------- | --------------------- |
+| chunk   | U256 | Index of a chunk.|
+| poolKey | poolKey     | The pool key of a specified pool.|
+
+#### Output parameters
+
+| Type | Description                 |
+| ---- | --------------------------- |
+| U256 | The chunk's bits. |
+
+### Flip bit utility
+
+```rust
+fn flipBitAtPosition(value: U256, position: U256) -> U256;
+```
+
+Flips bit in value at position.
+
+| Name  | Type    | Description           |
+| ----- | ------- | --------------------- |
+| value   | U256 | The value in which we flip a bit.|
+| position | U256     | The position of a bit to set.|
+
+#### Output parameters
+
+| Type | Description                 |
+| ---- | --------------------------- |
+| U256 | `value` with the specified bit flipped. |
+
+### Contains initialized tick
+
+```rust
+pub fn getBit(tick: I256, poolKey: PoolKey) -> Bool;
+```
+
+Retrieves the state of the exact bit representing the initialization state of a tick (1 - initialized, 0 - uninitialized).
+
+
+#### Input parameters
+
+| Name  | Type    | Description           |
+| ----- | ------- | --------------------- |
+| tick   | I256 | Index of a tick.|
+| poolKey | poolKey     | The pool key of a specified pool.|
+
+#### Output parameters
+
+| Type | Description                 |
+| ---- | --------------------------- |
+| Bool | State of the tick, `true` if initialized. |
+
+## Reserves
+
+```rust
+Abstract Contract Reserves() {...}
+
+Contract Invariant(...) extends Reserves(), ...{
+    ...
+        // Key: TokenId, Value: ReserveId
```

Comment:
Either we mention the key and value in every place or nowhere

Reply:
Nowhere